nvdaHelper/vbufBackends/mshtml/node.h

```diff
@@ -32,12 +32,13 @@ class MshtmlVBufStorage_controlFieldNode_t : public VBufStorage_controlFieldNode
 	DWORD HTMLChangeSinkCookey;
 	std::wstring language;
 	VBufStorage_controlFieldNode_t*  ariaLiveNode;
+	const wchar_t* ariaLiveLevel;

```

Comment:
What is the lifetime of this? When does it get cleaned up? Currently this seems to rely on `attribsMap` not changing since it points to internal data.

Reply:
I now changed this to a std::wstring to be more consistent with other strings on the node. Does that solve your concern as well?

Reply:
Yes, I beleive this is resolved.